src/debug/ee/wks/CMakeLists.txt

```diff
@@ -15,7 +15,11 @@ add_custom_command(OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/dbghelpers.obj
 #mark obj as source that does not require compile
 set_source_files_properties(${CMAKE_CURRENT_BINARY_DIR}/dbghelpers.obj PROPERTIES EXTERNAL_OBJECT TRUE)
```

Comment:
It looks this should be moved next to the add_library inside the if since the obj is not used for ARM64 now.